App.js

```diff
@@ -8,11 +8,17 @@ import { createStackNavigator } from '@react-navigation/stack';
 import TripPreview from './Components/TripPreview/TripPreview';
 import DatePicker from './Components/DatePicker/DatePicker';
 import Maps from './Components/Maps/Maps';
-import AddDestinations from './Components/AddDestinations/AddDestinations'
-import LandingPage from './Components/LandingPage/LandingPage'
+import AddDestinations from './Components/AddDestinations/AddDestinations';
+import LandingPage from './Components/LandingPage/LandingPage';
+import { YellowBox } from 'react-native';
 
 export const App = ({ navigation }) => {
 
+  YellowBox.ignoreWarnings([
```

Comment:
Still getting a yellow warning about the serializable values.  We may have to change where we call this.